content_manager/blocks/sections.py

```diff
@@ -0,0 +1,355 @@
+        return forms.Media(
+            css={"all": ("css/admin-block/cta-section-grid-block-admin.css",)},
+        )
+
+
+register(CTAGridAdapter(), CTASection)
+
+
+class SpotLightItem(blocks.StreamBlock):
+    card = VerticalCardBlock()
+
+    def get_default(self):
+        image_placeholder = Image.objects.filter(title="Placeholder Sites Faciles").first()
+
+        default_card_data = {
+            "title": "Titre de l'article",
+            "heading_tag": "h3",
+            "description": (
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, "
+                "sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. "
+                "Ut enim ad minim veniam, quis nostrud exercitation ullamco "
+            ),
+            "image": {
+                "image": image_placeholder,
+                "decorative": True,
+                "alt_text": "",
+            },
+            "image_ratio": "h3",
+            "image_badge": [],
+            "link": {
+                "link_type": "--",
+                "page": None,
+                "external_url": "",
+                "document": None,
+                "anchor": "",
+            },
+            "top_detail_text": "",
+            "top_detail_icon": {},
+            "top_detail_badges_tags": TEMPLATE_EXAMPLE_TAG_BADGE_LIST,
+            "bottom_detail_text": "",
+            "bottom_detail_icon": {},
+            "call_to_action": [],
+            "grey_background": False,
+            "no_background": False,
+            "no_border": False,
+            "shadow": False,
+        }
+        return [
+            ("card", default_card_data),
+            ("card", default_card_data),
+            ("card", default_card_data),
+        ]
+
+
+class SpotlightSection(BaseSection):
+    items_per_row = blocks.ChoiceBlock(
+        choices=[("6", "2"), ("4", "3"), ("3", "4")], default="4", label=_("Items per row")
+    )
+    link = LinkBlock(
+        label=_("Section link"),
+        help_text="This link appears at the bottom left of the section if completed",
+        required=False,
+        collapsed=True,
+    )
+    items = SpotLightItem(label=_("Items"))
+
+    class Meta:
+        template = "content_manager/blocks/sections/spotlight.html"
+        form_classname = "struct-block spotlight-grid-block"
+
+
+class SpotlightSectionGridAdapter(StructBlockAdapter):
+    """
+    Adapter to add the styling to the admin form
+    """
+
+    @cached_property
+    def media(self):
+        return forms.Media(
+            css={"all": ("css/admin-block/spotlight-grid-block-admin.css",)},
+        )
+
+
+register(SpotlightSectionGridAdapter(), SpotlightSection)
+
+
+def get_accordion_default():
+    return [
+        ("title", "Titre de section"),
+        ("accordion", {"title": "Titre de l'accordéon 1", "content": "<p>Contenu de l'accordéon 1</p>"}),
+        ("accordion", {"title": "Titre de l'accordéon 2", "content": "<p>Contenu de l'accordéon 2</p>"}),
```

Comment:
```suggestion
        ("accordion", {"title": "Titre de l'accordéon 2", "content": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>"}),
```
Je pense qu'il faut un peu de longueur de texte dans les accordéons donc je suggère du lorem ipsum ici